CDS-74532 Helm Chart expressions added (#3113)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/kubernetes-services.md

Harness supports the following manifest types and orchestration methods.

> For more details and options when using Kubernetes manifests in Harness, go to [Add Kubernetes manifests](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-kubernetes-category/define-kubernetes-manifests).

Reviewer: ```suggestion
For more information about using Kubernetes manifests in Harness, go to [Add Kubernetes manifests](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-kubernetes-category/define-kubernetes-manifests).
```
